requirements/readthedocs.txt

```diff
@@ -0,0 +1,4 @@
+mmcv
```

Comment:
Shall we merge `docs.txt` into this one? Unless there's a situation that we only have to install one of those.

Reply:
`requirements/readthedocs.txt` is mainly used in `.readthedocs.yml` for readthedoc website to build the repo. For `docs.txt`, it is also used to build repo locally, where we may already install the necessary packages like mmcv.

Reply:
Still wondering that if there's anyone who wants to build it locally. We could have a discussion later.

Reply:
Offline discussion sounds fair, question resolved.